src/java/org/apache/cassandra/tools/nodetool/ListSnapshots.java

```diff
@@ -50,6 +55,7 @@ public void execute(NodeProbe probe)
 
             Map<String, String> options = new HashMap<>();
             options.put("no_ttl", Boolean.toString(noTTL));
+            options.put("no_ephemeral", Boolean.toString(!ephemeral));
```

Comment:
It doesn't feel natural passing `!ephemeral`. Since the default is to **NOT** show snapshots, the property name should be called `include_ephemeral`, which defaults to false - and we can pass the `ephemeral` field without negating it.